test/integration/http2_integration_test.cc

```diff
@@ -28,12 +28,41 @@ INSTANTIATE_TEST_SUITE_P(IpVersions, Http2IntegrationTest,
                          TestUtility::ipTestParamsToString);
 
 TEST_P(Http2IntegrationTest, RouterRequestAndResponseWithBodyNoBuffer) {
-  testRouterRequestAndResponseWithBody(1024, 512, false);
+  testRouterRequestAndResponseWithBody(1024, 512, false, false);
```

Comment:
The growth of parameters is getting a bit out of hand in some of these test calls, do you think it would make sense to add an options struct?

Reply:
Yes, but in a different PR.

Reply:
OK, would be a great followup PR, thanks.